server/util.go

```diff
@@ -33,14 +34,21 @@ import (
 	"go.uber.org/zap"
 )
 
-// CheckPDVersion checks if PD needs to be upgraded.
-func CheckPDVersion(opt *config.PersistOptions) {
+// CheckAndGetPDVersion checks and returns the PD version.
+func CheckAndGetPDVersion() *semver.Version {
 	pdVersion := versioninfo.MinSupportedVersion(versioninfo.Base)
 	if versioninfo.PDReleaseVersion != "None" {
 		pdVersion = versioninfo.MustParseVersion(versioninfo.PDReleaseVersion)
 	}
+	return pdVersion
+}
+
+// CheckPDVersionWithClusterVersion checks if PD needs to be upgraded by comparing the PD version with the cluster version.
+func CheckPDVersionWithClusterVersion(opt *config.PersistOptions) {
```

Comment:
Can we remove the `CheckAndGetPDVersion` function by passing `GetClusterVersion`?

Reply:
Oh, for the first time, it seems no need to check cluster version 0 0